docs/sources/developers/plugins/plugin.schema.json

```diff
@@ -7,6 +7,11 @@
   "required": ["type", "name", "id", "info", "dependencies"],
   "additionalProperties": false,
   "properties": {
+    "$schema": {
+        "type": "string",
+        "description": "Schema definition for the plugin.json file",
+        "enum": ["https://raw.githubusercontent.com/grafana/grafana/master/docs/sources/developers/plugins/plugin.schema.json"]
```

Comment:
I like the idea of being able to auto-complete the plugin.json, but I'd ideally like to have a more stable URL. Should we consider publishing the schema to https://www.schemastore.org/json/, i.e. `https://json.schemastore.org/grafana-plugin-metadata`?

Reply:
Ideally this should be published to schema store where the IDEs will search for schema. But the source should be with grafana repo. Since this not a mandatory fields, the enum can be enhanced as we progress. 

Below example shows VS code search for schema list from schema store.

![image](https://user-images.githubusercontent.com/153843/90423974-47693100-e0b5-11ea-8a36-f37b7a4c55b6.png)

Reply:
Maybe for now, we can remove the `enum` and allow any string until we have a stable URL? I'd like to open a PR to the schemastore once this has been merged.

Reply:
Done marcus